java/query/query-lambdav2/src/test/java/sleeper/query/lambda/SqsQueryProcessorLambdaIT.java

```diff
@@ -0,0 +1,818 @@
+        LocalDate startDate = LocalDate.of(minYear, 1, 1);
+        LocalDate endDate = LocalDate.of(maxYear + 1, 1, 1);
+        List<Record> records = new ArrayList<>();
+        for (LocalDate date = startDate; date.isBefore(endDate); date = date.plusDays(1)) {
+            Record record = new Record();
+            record.put("year", date.getYear());
+            record.put("month", date.getMonthValue());
+            record.put("day", date.getDayOfMonth());
+            record.put("timestamp", Date.from(Timestamp.valueOf(date.atStartOfDay()).toInstant()).getTime());
+            record.put("count", (long) date.getYear() * (long) date.getMonthValue() * (long) date.getDayOfMonth());
+            HashMap<String, String> map = new HashMap<>();
+            map.put(date.getMonth().name(), date.getMonth().name());
+            record.put("map", map);
+            record.put("list", Lists.newArrayList(date.getEra().toString()));
+            record.put("str", date.toString());
+            records.add(record);
+        }
+
+        return records;
+    }
+
+    private TableProperties createTimeSeriesTable(Integer minSplitPoint, Integer maxSplitPoint) {
+        List<Object> splitPoints = new ArrayList<>();
+        for (int i = minSplitPoint; i <= maxSplitPoint; i++) {
+            splitPoints.add(i);
+        }
+
+        return createTimeSeriesTable(splitPoints);
+    }
+
+    private TableProperties createTimeSeriesTable(List<Object> splitPoints) {
+        TableProperties tableProperties = createTestTableProperties(instanceProperties, SCHEMA);
+        S3TableProperties.createStore(instanceProperties, s3ClientV2, dynamoClientV2).save(tableProperties);
+
+        StateStore stateStore = new StateStoreFactory(instanceProperties, s3ClientV2, dynamoClientV2, s3TransferManager)
+                .getStateStore(tableProperties);
+        try {
+            update(stateStore).initialise(new PartitionsFromSplitPoints(tableProperties.getSchema(), splitPoints).construct());
+        } catch (StateStoreException e) {
+            throw new RuntimeException(e);
+        }
+
+        return tableProperties;
+    }
+
+    private InstanceProperties createInstance(String dir) {
+        InstanceProperties instanceProperties = createTestInstanceProperties();
+        instanceProperties.set(FILE_SYSTEM, "file://");
+        instanceProperties.set(DATA_BUCKET, dir);
+        instanceProperties.set(DEFAULT_INGEST_PARTITION_FILE_WRITER_TYPE, "direct");
+
+        new DynamoDBQueryTrackerCreator(instanceProperties, dynamoClientV2).create();
+
+        instanceProperties.set(QUERY_QUEUE_URL, sqsClient.createQueue(UUID.randomUUID().toString()).getQueueUrl());
+        instanceProperties.set(LEAF_PARTITION_QUERY_QUEUE_URL, sqsClient.createQueue(UUID.randomUUID().toString()).getQueueUrl());
+        instanceProperties.set(QUERY_RESULTS_QUEUE_URL, sqsClient.createQueue(UUID.randomUUID().toString()).getQueueUrl());
```

Comment:
This is still using AWS SDK v1. If you use the helper in LocalStackTestBase to create the queues, that can be migrated separately.